Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Service/ReportService.cs

```diff
@@ -0,0 +1,355 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using Newtonsoft.Json;
+using Sdl.Community.ExportAnalysisReports.Helpers;
+using Sdl.Community.ExportAnalysisReports.Interfaces;
+using Sdl.Community.ExportAnalysisReports.Model;
+using Sdl.ProjectAutomation.Core;
+
+namespace Sdl.Community.ExportAnalysisReports.Service
+{
+	public class ReportService : IReportService
+	{
+		public static readonly Log Log = Log.Instance;
+		private readonly string _communityFolderPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "SDL Community", "ExportAnalysisReports");
+		private readonly Help _help;
+		private readonly IMessageBoxService _messageBoxService;
+		private readonly IStudioService _studioService;
+
+		public ReportService(IMessageBoxService messageBoxService, IStudioService studioService)
+		{
+			_help = new Help();
+			_messageBoxService = messageBoxService;
+			_studioService = studioService;
+		}
+
+		public string JsonPath => Path.Combine(_communityFolderPath, "ExportAnalysisReportSettings.json");
+		public string ReportsFolderPath { get; set; }
+		/// <summary>
+		/// Check if the report was successfully generated
+		/// </summary>
+		/// <param name="reportOutputPath"></param>
+		/// <param name="isChecked"></param>
+		/// <param name="optionalInformation"></param>
+		/// <param name="projects"></param>
+		/// <returns></returns>
+		public bool GenerateReportFile(BindingList<ProjectDetails> projects, OptionalInformation optionalInformation, string reportOutputPath, bool isChecked)
+		{
+			try
+			{
+				_help.CreateDirectory(reportOutputPath);
```

Comment:
This could be done here directly since it's only a line of code.

Reply:
Indeed, i've replace _help.CreateDirectory(reportOutputPath); with Directory.CreateDirectory(reportOutputPath); and remove the method CreateDirectory(string path) from Help class. In this way i've got rid of unnecessary references to Help class inside the ReportsService.
The changes will be reflected within the next commit